### ISSUE TITLE
Add Google Summer of Code to the programs section on the website

### ISSUE THREAD

**Issue by palash25** [opensourcediversity/opensourcediversity.org#25]
I know that the list focusses on SOC programs for women and other underrepresented sections in software development but the number of girls selected for Google Summer of Code has been increasing over the years and I think more girls should know about such a program.

**Comment by jancborchardt** [comment]
What do you think @jonatoni @camilasan @kristiprogri @Incabell and others? Was a bit hesitant just because the Google name is attached to it, but it is a pretty popular and fitting program though. On the other hand, Outreachy and RailsGirls Summer of Code was started precisely _because_ Google Summer of Code was/is not particularly diverse.

**Comment by Incabell** [comment]
hm, I'm not sure either. on one hand, we want to show all inclusionist places, on the other it's not specifically there to aid inclusion but is naturally being inclusionist, which is great, but might be confusing since I'd expect the former when looking at the site. (not sure how understandable that sentence is, so please ask if it's unclear what I mean). 

how would we determine, if a project is being good at including underrepresented groups and when it isn't? what if it stops trying? do we take it off the list? to we check every year if things are still the same?

I think my gut feeling is telling me, that projects that are not especially aimed at increasing diversity but still try to do so (which is amazing) should get a separate section or page where they can be found but shouldn't be amongst the other projects, otherwise it'll get very confusing very fast.

I'm totally up for discussion about this since I'm not super certain that is the best way to go about it and I'll happily be convinced otherwise, but that's my gut feeling as of now.

**Comment by palash25** [comment]
> I think my gut feeling is telling me, that projects that are not especially aimed at increasing diversity but still try to do so (which is amazing) should get a separate section or page where they can be found but shouldn't be amongst the other projects, otherwise it'll get very confusing very fast.

I agree a separate section for such programs like GSoC, Linux Foundation Internships would be nice. It can also be mentioned that they are not exclusive to women but should give it a try in order to increase the diversity in these projects too.

Just because they are not exclusive to women shouldn't stop them from trying. I think a place on this website might even encourage a few of them to try thereby increasing the diversity.

**Comment by palash25** [comment]
Here are a few more programs that can be added to a separate section if you folks are interested.
https://github.com/tapasweni-pathak/SOC-Programs#soc-programs

**Comment by jonatoni** [comment]
I'm +1 to add GSoC too. Why? - because GSoC is a global program focused on open source software development and helps students to get more involved with open source communities. Don't forget that during the application period we encourage people to apply at both internships and check both of them when they apply because of the diverse projects (and project ideas) they can apply. Yes, it is focused only on students that like coding, but they can be beginners or not, they can get involved with Python/Rails/JS/C++ etc so we have different ways someone can get involved and start contributing to open source. I would be skeptic if the project is related to tech in general and not specifically with open source.

**Comment by Shelly011s** [comment]
Is this issue still open?

**Comment by jwflory** [comment]
Hey, as a former GSoC student and mentor, I am +0 to adding Google. Honestly I want to give this a -1.

I am opposed to adding GSoC because interns do not have support systems for DEI issues in the communities they work in. GSoC interns are essentially contractors paid by Google to work in the open source communities that apply to GSoC. GSoC mentors can also be org admins, which can (and has before) cause conflicts of interest, especially when GSoC is used as a proxy recruiting program for new talent. I have seen a GSoC community where the org admin and mentor sexually harassed an intern. I am adamant that GSoC needs a stronger safety net to protect interns from manipulative and/or abusive mentors. So, as a rule of thumb, I generally avoid recommending it to FOSS newcomers, especially if they are from an underrepresented group in tech.

That said, I know many communities who participate in GSoC are good projects and communities. I am also confident the number of good communities outnumbers the list of bad communities. If others feel strongly, then we can promote GSoC as a program for creating opportunities for others where it was not before. But maybe we can feature it… less prominently? Or just not use the Google logo on the page.
